src/psyclone/parse/module_info.py

```diff
@@ -92,6 +103,44 @@ def __init__(self, name, filename):
         # each module, indexed by the module names: Dict[str, Set(str)].
         self._used_symbols_from_module = None
```

Comment:
No this PR but s/None/{}/ as it's a dict?

Reply:
I am using this to distinguish between 'not yet filled' (None) and 'no symbols' ({}).